docs/getting-started/uninstall_turtles.md

```diff
@@ -14,11 +14,11 @@ To simplify uninstalling Rancher Turtles (via Rancher Manager or helm command),
 - Delete the CAPI `deployments` that are no longer needed.
 :::
 
-There are two options to uninstall the Rancher Turtles Operator depending on the installation method.
+There are two options to uninstall the Rancher Turtles depending on the installation method.
```

Comment:
As discussed we will only mention uninstalling via helm on the cli initially. But this can be changed in #77. wdyt @furkatgofurov7 @Danil-Grigorev?

Reply:
Let's change it in #77 👍🏼